datanode/api/trading_data_v2.go

```diff
@@ -1325,6 +1325,10 @@ func (t *TradingDataServiceV2) sendPositionsSnapshot(ctx context.Context, req *v
 func (t *TradingDataServiceV2) GetParty(ctx context.Context, req *v2.GetPartyRequest) (*v2.GetPartyResponse, error) {
 	defer metrics.StartAPIRequestAndTimeGRPC("GetParty")()
 
+	if len(req.PartyId) == 0 {
+		return nil, formatE(ErrMissingPartyID)
+	}
```

Comment:
For all these check for the IDs being here or not, would you also check if they are well formed?
Using crypto.IsValidVegaPubkey or crypto.IsValidVegaID?

Also for parties, "network" is a valid ID as an exception.

Reply:
Resolved https://github.com/vegaprotocol/vega/pull/8070/commits/c16b06e4dcbd9087c4ee1fc4790d58f4f5c2eb71 and https://github.com/vegaprotocol/vega/pull/8070/commits/7c776bc1d6960656dfbad71ef5f7a9eece986e97